### DISCUSSION TITLE
gRPC mock - How can I log or view the incoming message

### DISCUSSION THREAD

**Discussion by tpejr** [fauxauldrich/camouflage#148] [Q&A] [answered]
I am creating an gRPC mock simulation config and would like to know how to see the entire incoming message so I can check it's structure.

**Comment by shubhendumadhukar** [comment, marked as answer]
For client/bidi streams I believe enabling debug logs would show the incoming request body. I have added some more debugs to log the same for unary and server streams. [67a1ca8](https://github.com/testinggospels/camouflage/commit/67a1ca8088abe2679033bb230b25b3a1a473d530)

I hope this might help you visualise your request better.

**Reply by tpejr** [reply]
Thankyou @shubhendumadhukar , the debug logging for unary requests is what I need!   I apologize as I  should have been more specific in my question.
